components/faqs/support-for-css-frameworks.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+
+const sampleTachyonsExample = (`
+const Button = styled.button.attrs({
+  className: 'pa4 ba b--black-80',
```

Comment:
Like I'm saying below: Please don't make it tachyons specific and feel free to add 1-3 example classes to our global css (_document.js) to make this example work.